capacity4more/behat/features/homepage.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Test homepage activity stream
```

Comment:
There are 2 features for homepage:

* homepage_anonymous.feature
* homepage_logged_in.feature